src/path_rewriting.rs

```diff
@@ -280,78 +272,76 @@ pub fn rewrite_paths(
         }
     }
 
-    let mut cache: Option<PathBuf> = None;
```

Comment:
Is removing this case correc? It seems as though it was tied to the in-order evaluation of files (which no longer applies because parallel is by nature unordered).